### DISCUSSION TITLE
I can't get past the verification stage when re-applying for GitHub student.

### DISCUSSION THREAD

**Discussion by oscie57** [community/community#54599] [GitHub Education]
### Hi everyone,

I previously had the GitHub student pack, and I noticed the other day that I was missing some of my benefits. Upon opening the GitHub education page, I noticed I couldn't access the benefits page, and that I have to re-apply. This makes sense, as it does say that I may occasionally have to prove I go there, and it has been 2 years since I last did.

When uploading my School ID, there is a message at the bottom saying the following:

> Your application cannot be reviewed until you fix the following:
> - You have an outstanding discount request for @oscie57. We will get back to you soon.

I have no idea what this means, as I have not received an email about any of this.

**Comment by hrishikesh-kocharekar-dev** [comment]
hi Oscar,
It's possible that the outstanding discount request for @oscie57 refers to someone else who may have used your email or GitHub username to apply for the GitHub Student Developer Pack. It's also possible that there is a mix-up with your application and someone else's.

You should reach out to the GitHub Education Support team to inquire about this issue. You can do this by going to the GitHub Education page, scrolling down to the bottom, and clicking on "Contact Us" under "Still have questions?". From there, you can explain the situation and ask for their assistance in resolving the issue.

**Comment by muhiris** [comment]
I'm also getting the same issue, even I'm  a stuent.

![image](https://github.com/community/community/assets/80543033/a6cd24af-a14e-4a61-8c2f-4b036b0b98a7)

**Reply by OYEFAVOUR** [reply]
Go to your application and on the page where you have the error, click "Cancel", then re-apply. That worked for me.

**Reply by samus-aran** [reply]
Awesome answer @OYEFAVOUR. ❤️ Something extra to note is that an application that is in draft mode for a number of days it will cancel itself.

**Reply by mehedihjim** [reply]
Thanks man
